test/models/coupon_test.rb

```diff
@@ -1,7 +1,22 @@
 require 'test_helper'
 
 class CouponTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'coupon in valid state' do
+    coupon = create(:coupon)
```

Comment:
The way you wrote this is fine, just wanted to show how we do it. We normally do
```
coupon = build(:coupon)
assert coupon.save
```